src/pydap/net.py

```diff
@@ -34,7 +34,7 @@ def raise_for_status(response):
 def follow_redirect(url, application=None, session=None):
     """
     This function essentially performs the following command:
-    >>> Request.blank(url).get_response(application)
+    >>> Request.blank(url).get_response(application) #doctest: +SKIP
```

Comment:
PEP8 guidelines for inline comments are to always preface them by at least two lines and include a space after `#`. So this should be:
```
>>> Request.blank(url).get_response(application)  # doctest: +SKIP
```